ngraph/frontend/generic/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cc)
```

Comment:
You don't have `.cc` files (the same as `.h`)

Reply:
Fixed